custom/icds_reports/views.py

```diff
@@ -1730,7 +1730,7 @@ def valid_state_names(self):
 class CasDataExport(View):
     def post(self, request, *args, **kwargs):
         data_type = int(request.POST.get('indicator', None))
-        state_id = request.GET.get('location', None)
+        state_id = request.POST.get('location', None)
```

Comment:
more of a correct class attribute instead of method?

Reply:
I meant http method